app/channels/application_cable/channel.rb

```diff
@@ -0,0 +1,4 @@
+module ApplicationCable
+  class Channel < ActionCable::Channel::Base
+  end
+end
```

Comment:
I dont remember this in past projects
Do we need this?

Reply:
I guess we do